code/modules/mob/living/living.dm

```diff
@@ -278,17 +278,19 @@
 
 /mob/living/resist_grab(moving_resist)
 	if(pulledby.grab_level)
-		if(prob(30/pulledby.grab_level))
+		grab_resist_level += 1
```

Comment:
This doesn't get reset unless by breaking a grab. Means I can store grab resists to be used later in combat.

Reply:
Looks like you're right.

1. Have someone get the strongest grab on you.
2. Resist until you're one resist away from breaking free.
3. Get grabbed by a xeno, resist once and you're free.
4. ???
5. Profit

Reply:
This has been fixed by resets to grab_resist_level elsewhere